lib/config/presets/internal/default.ts

```diff
@@ -112,6 +112,12 @@ export const presets: Record<string, Preset> = {
     dependencyDashboardApproval: true,
     description: 'Enable Renovate Dependency Dashboard approval workflow.',
   },
+  dependencyDashboardApprovalForMajor: {
```

Comment:
I'm looking for feedback on the:

- name of the preset
- correct location of this file
- syntax of the new code